packages/netlify/test/functions/redirects.test.js

```diff
@@ -13,8 +14,9 @@ describe('SSR - Redirects', () => {
 	it('Creates a redirects file', async () => {
 		const redirects = await fixture.readFile('./_redirects');
 		const parts = redirects.split(/\s+/);
-		expect(parts).to.deep.equal(['', '/other', '/', '301', '']);
-		expect(redirects).to.matchSnapshot();
+		assert.deepEqual(parts,['', '/other', '/', '301', '']);
+		// Snapshots are not supported in Node.js test yet (https://github.com/nodejs/node/issues/48260)
+		assert.equal(redirects,'\n/other    /       301\n');
```

Comment:
There might be a file left on disk that contains the (now unused) snapshot contents. Should we remove that?

Reply:
If there are any files which we don't need anymore, we should remove it!